app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -129,17 +144,14 @@ class ToolDetailsFragment() :
 
     fun openToolTraining(tool: Tool?, translation: Translation?) {
         val code = tool?.code ?: return
-        val locale = translation?.languageCode ?: return
+        translation?.languageCode ?: return
```

Comment:
you should still assign the locale to a local property and then just use the local property